lib/v2/cankaoxiaoxi/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/news/:category?', require('./index'));
```

Comment:
```suggestion
    router.get('/news/:category', require('./index'));
```
`category` looks like a compulsory parameter according to your code